scripts/branch_rename_migration.py

```
@@ -0,0 +1,251 @@
+- refactor/short-description
+- docs/short-description
+"""
+
+import subprocess
+import sys
+import re
+
+def run_command(command):
+    """Run a shell command and return the output."""
+    try:
+        result = subprocess.run(command, shell=True, capture_output=True, text=True, check=True)
```

Ruff (0.14.4)

`subprocess` call with `shell=True` identified, security issue

(S602)